.travis.yml

```diff
@@ -13,20 +13,22 @@ jobs:
       script: sudo ./tools/travis/run_tests.sh static
     - stage: unit
       if: type != cron
-      script: sudo ./tools/travis/run_tests.sh unit
+      script: sudo ./tools/travis/run_tests.sh snapcraft/tests/unit
```

Comment:
Is everything nested within `snapcraft/tests/`? If so, would it be worth making it so that `snapcraft/tests/` doesn't need to be specified each time?

Reply:
@kyrofa I could preppend snapcraft/tests if the argument doesn't start with that.
But it worries me that we might fall in the same trap as before, where the script is some dark magic. It seemed to me that using the full path would make it clearer if you want to filter and run only one subdirectory, which before had to do with a weird thing like `*python*`.

No strong opinion though, I would say it's a trivial change, but bash is never trivial. Let's say "easy".

Reply:
Haha, no that's fine.